data_steward/analytics/table_metrics/metrics_over_time/dictionaries_and_lists.py

```diff
@@ -275,7 +275,8 @@
     "cpmc_ucd": "UC Davis",
     "ipmc_rush": "Rush University",
     "va": "United States Department of Veterans Affairs - Boston",
-    "saou_umc": "University Medical Center (UA Tuscaloosa)"
+    "saou_umc": "University Medical Center (UA Tuscaloosa)",
+    "saou_usahs": "University of South Alabama"
```

Comment:
All hpo-related items need to use the lookup table. We shouldn't need to update a new site in multiple places

Reply:
@nishanthpp93 
This script actually does not use BigQuery at all (it is intended to be used on the reports that I run every week) so we cannot use a lookup table unless we integrate this into the pipeline.

Reply:
I see. Can this still use BigQuery without being integrated into the pipeline?

Reply:
@nishanthpp93 Potentially but that would be a big change - something that would warrant its own ticket and need to be tested thoroughly. I can make a ticket in EDQ to investigate whether or not that is feasible.

Reply:
We shouldn't be manually updating sites in different places because it's easy to miss them. Keep me in the loop on the ticket as well. I'll merge this for now.

Reply:
@nishanthpp93 I documented this in EDQ-534. It is low priority at the moment (see the 'NOTE') so it will fall under the responsibilities of my successors.